src/openlifu/trk/photoscan.py

```diff
@@ -320,3 +359,136 @@ def merge_textures(input_obj_path: Path, output_path: Path) -> None:
     mesh.visual = trimesh.visual.TextureVisuals(uv=new_uvs, image=new_texture)
 
     mesh.export(output_path / "texturedMesh.obj")
+
+
+def apply_exif_orientation_numpy(image: np.ndarray, orientation: int, inverse: bool = False) -> np.ndarray:
+    """
+    Transforms an image array to undo or redo EXIF orientation.
+
+    Args:
+        image (np.ndarray): The image array (H x W x C or H x W).
+        orientation (int): EXIF orientation tag value (1-8).
+        inverse (bool): If True, applies the inverse transformation.
+
+    Returns:
+        np.ndarray: The transformed image.
+    """
+    if orientation == 1:
+        return image  # No transformation needed
+
+    # Define forward transformations
+    def transform(img):
+        if orientation == 2:
+            return np.fliplr(img)
+        elif orientation == 3:
+            return np.rot90(img, 2)
+        elif orientation == 4:
+            return np.flipud(img)
+        elif orientation == 5:
+            return np.rot90(np.fliplr(img), -1)
+        elif orientation == 6:
+            return np.rot90(img, -1)
+        elif orientation == 7:
+            return np.rot90(np.fliplr(img), 1)
+        elif orientation == 8:
+            return np.rot90(img, 1)
+        else:
+            raise ValueError("Unknown exif orientation")
+
+    # Define inverse transformations
+    def inverse_transform(img):
+        if orientation == 2:
+            return np.fliplr(img)
+        elif orientation == 3:
+            return np.rot90(img, 2)
+        elif orientation == 4:
+            return np.flipud(img)
+        elif orientation == 5:
+            return np.fliplr(np.rot90(img, 1))
+        elif orientation == 6:
+            return np.rot90(img, 1)
+        elif orientation == 7:
+            return np.fliplr(np.rot90(img, -1))
+        elif orientation == 8:
+            return np.rot90(img, -1)
+        else:
+            raise ValueError("Unknown exif orientation")
+
+    return inverse_transform(image) if inverse else transform(image)
+
+
+def get_modnet_path():
+    """Get the MODNet checkpoint path. Download it if not present.
+    """
+    package = "openlifu.trk.modnet_checkpoints"
+    filename = "modnet_photographic_portrait_matting.onnx"
```

Comment:
let's add `src/openlifu/trk/modnet_checkpoints/modnet_photographic_portrait_matting.onnx` to the .gitignore at the top level of this repo

or maybe more generally just `*.onnx`?